components/eam/src/physics/cam/clubb_intr.F90

```diff
@@ -2036,6 +2053,15 @@ subroutine clubb_tend_cam( &
 
       !  Surface fluxes provided by host model
       wpthlp_sfc = real(cam_in%shf(i), kind = core_rknd)/(real(cpair, kind = core_rknd)*rho_ds_zm(1)) ! Sensible heat flux
+#if 1
+      inv_exner_clubb_surf = 1._r8/((state1%pmid(i,pver)/p0_clubb)**(rair/cpair)) !phl Option 2
+      wpthlp_sfc = wpthlp_sfc*inv_exner_clubb_surf
+#endif
+#if 0
+      inv_exner_clubb_surf = 1._r8/((state1%pint(i,pverp)/p0_clubb)**(rair/cpair)) !Peter B option
+      wpthlp_sfc = wpthlp_sfc*inv_exner_clubb_surf
```

Comment:
@oksanaguba, could you link to where this is implemented in CAM or CESM?  I know that NCAR talked about changing this, but I can't find the change in their code.

Reply:
Adam gave us a link (not sure, is it ok to share here?), there are many other changes in there as well.

Reply:
@oksanaguba please share.  If its CESM's github, its ok.

Reply:
@rljacob  https://github.com/ESCOMP/CAM/pull/598

Reply:
@oksanaguba , any reference that surface sensible heat flux in E3SM is formulated proportional to w'T'_sfc? To my knowledge, it has been in w'theta'_sfc already (say [over land here from CAM6 doc)](https://ncar.github.io/CAM/doc/build/html/cam6_scientific_guide/physics.html#surface-exchange-formulations)

Reply:
I think that @wlin7 asks the key question: is the coupler in E3SM delivering a surface sensible heat flux from the land or ocean model in terms of temperature (w'T') or potential temperature (w'theta')?  I am not familiar with E3SM's land model, but CLUBB conserves theta_l, not absolute temperature.  

CESM's land model appears to be delivering w'T', based on Fig. 12 of [Lauritzen et al. (2022)](https://agupubs.onlinelibrary.wiley.com/doi/pdf/10.1029/2022MS003117).  Fig. 12 shows that if the flux from the coupler is divided by the Exner function, then the energy conservation is better over mountainous regions.  If DOE sees the same pattern, then that suggests that the fix is implemented correctly.